src/main/java/net/sourceforge/argparse4j/inf/ArgumentParserException.java

```diff
@@ -23,6 +23,9 @@
  */
 package net.sourceforge.argparse4j.inf;
 
+import static net.sourceforge.argparse4j.internal.MessageLocalization.localizeIfPossible;
```

Comment:
Let me know if I can help.

Personally I am not too bothered by cyclic dependencies within in the same module. `internal` contains the implementation code that is shared by types in multiple public packages. You do not want to make package private copies of that implementation code in each public package, so you need an additional package to put the code in. Logically this implementation package will need to refer to the types in the public packages too, so cycles are sort of inevitable.

I am sure that it is possible to create a package structure where cycles do not occur, but in my opinion the advantages of such as structure do not outweigh the possibly strange code constructs needed to achieve it.